### ISSUE TITLE
A11y_Azure Tools Storage Explorer_Get Shared Access Signature_Use of Color : The "Learn More about shared access signature" is not associated with underline/chevron to differentiate as link from normal text on dialog.

### ISSUE THREAD

**Issue by keerthiduvvuri** [microsoft/AzureStorageExplorer#7595]
### Preflight Checklist

- [X] I have installed the [latest version of Storage Explorer](https://github.com/Microsoft/AzureStorageExplorer/releases/latest).
- [X] I have checked existing resources, including the [troubleshooting guide](https://docs.microsoft.com/azure/storage/common/storage-explorer-troubleshooting) and the [release notes](https://github.com/microsoft/AzureStorageExplorer/releases).
- [X] I have searched for [similar issues](https://github.com/Microsoft/AzureStorageExplorer/issues).

### Storage Explorer Version

1.32.1(96)

### Regression From

NO

### Architecture

x64

### Storage Explorer Build Number

20231114.10

### Platform

macOS

### OS Version

MAC Version 13.6.1

### GitHub Tags:
#A11yMAS;#A11yTCS;#MAC;#DesktopApp;#A11ySev3;#BM_AzureToolsStorageExplorer_MAC_Dec2023;#Azure Tools Storage Explorer;#FTP;#Voiceover;#WCAG1.4.1;#UseofColor;

### Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.32.1 (96)
Screen Reader: Voice Over

### Steps to Reproduce

1. Launch Storage Explorer
2. Tab till "Open Connect dialog" pane and Sign in to Azure.
3. Tab till "Manage Accounts" pane and Select the subscriptions you will use. Once selected, context menu on "Apply" button.
4. Tab till "Blob Containers" of the selected subscription, press context menu and hit enter on "Create Blob container" to Create a new Blob container.
5. Once completed, Tab till "Upload" button and Upload a blob/file by using "Upload Files" control.
6. Blob gets uploaded, press context menu button on blob and select "Get Shared Access Signature" control and hit enter.
7. Verify that the "Learn More about shared access signature" are associated with underline/chevron to differentiate as link from normal text on dialog or not.

### Actual Experience

The "Learn More about shared access signature" are not associated with underline/chevron to differentiate as link from normal text on dialog.
Note: Similar issue is observed in "Manage Access policies" scenario also.

### Expected Experience

The links on the page should have the underline or chevron associated with it, in-order to differentiate from normal text on page.

### User Impact:
Color blind users will face difficulties to identify and easily distinguish the links on page from normal text as it does not have any underline/chevron associated with it.

### Attachment:
<img width="1440" alt="A11y_Azure Tools Storage Explorer_Get Shared Access Signature_Use of Color" src="https://github.com/microsoft/AzureStorageExplorer/assets/98728920/fdce7676-9308-4460-a280-b5d33927adb4">

![A11y_Azure Tools Storage Explorer_Get Shared Access Signature_Use of Color](https://github.com/microsoft/AzureStorageExplorer/assets/98728920/5a0694b2-d182-46fe-ac25-c22db8eb2186)

**Comment by JasonYeMSFT** [comment]
Need to update ui-components and any packages using it for dialogs.

**Comment by JasonYeMSFT** [comment]
I added the underline on main.

**Comment by JasonYeMSFT** [comment]
@kupatkar99 You closed another almost identical bug 11 hours ago. Could you double check this bug using the most recent build? If this reproduces in the most recent build, please share its build number with me.

https://github.com/microsoft/AzureStorageExplorer/issues/7583

**Comment by kupatkar99** [comment]
Verified the issue in below environment:
Version: 1.33.0 (97)
AzCopy Version: 10.23.0
Platform: macOS
Architecture: arm64
Build Number: 20240220.6
Issue is fixed. Hence Closing the bug.  Please find added attachment for reference.
<img width="1440" alt="7595_Fixed" src="https://github.com/microsoft/AzureStorageExplorer/assets/98530069/3da10e77-bdd4-4d8e-be85-245333e3740e">
